ex03.ex

```diff
@@ -55,7 +55,17 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  # def odd_even . . . "your code"
+  def odd_even([]),         do: []
+  def odd_even([head|tail]) do
```

Comment:
## -1

I'd really like to see this without an if, and with additional functions:

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~